src/main/java/frc/robot/commands/magazine/RunMagazineForward.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands.magazine;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Magazine;
+
+public class RunMagazineForward extends CommandBase {
```

Comment:
This needs to be combined with the RunMagazineBackward command.  No need to have them separate